accounts-cluster-bench/src/main.rs

```diff
@@ -340,6 +367,57 @@ fn run_rpc_bench_loop(
             break;
         }
         match rpc_bench {
+            RpcBench::Block => {
+                let mut rpc_time = Measure::start("rpc-get-block");
+                let slot_height = slot_height_atom.load(Ordering::Relaxed);
+                match client.get_block_with_config(
+                    slot_height,
+                    RpcBlockConfig {
+                        commitment: Some(CommitmentConfig::confirmed()),
+                        ..Default::default()
+                    },
+                ) {
+                    Ok(_block) => {
+                        rpc_time.stop();
+                        stats.success += 1;
+                        stats.total_success_time_us += rpc_time.as_us();
+                    }
+                    Err(e) => {
+                        rpc_time.stop();
+                        stats.total_errors_time_us += rpc_time.as_us();
+                        stats.errors += 1;
+                        if last_error.elapsed().as_secs() > 2 {
+                            info!("get_block error: {:?}", e);
+                            last_error = Instant::now();
+                        }
+                    }
+                }
+            }
+            RpcBench::Blocks => {
+                static NUM_BLOCKS_TO_FETCH: u64 = 100;
```

Comment:
inb4 ‘magic number.’
![giphy-506291741](https://github.com/user-attachments/assets/2e83e1d8-d6a0-4740-9757-0c8e0abe005c)

Reply:
should we use the max allowed response size instead? for seniority of magic, ofc